conda/recipes/kvikio/meta.yaml

```
@@ -71,7 +71,7 @@ requirements:
     - cupy >=12.0.0
     - zarr
     # See https://github.com/zarr-developers/numcodecs/pull/475
-    - numcodecs !=0.12.0
+- numcodecs>=0.13.0,<0.14.0
```

Sorry lost some blank spaces in my last review by accident

```suggestion
    - numcodecs >=0.13.0,<0.14.0a0
```